_unittest_solvers/test_26_emit.py

```diff
@@ -694,16 +682,9 @@ def test_12_radio_getters(self, add_app):
     )
     def test_13_static_type_generation(self):
         domain = self.aedtapp.results.interaction_domain()
-        if sys.version_info < (3, 8):
-            py_version = "EmitApiPython"
-        elif sys.version_info < (3, 9):
-            py_version = "EmitApiPython38"
-        elif sys.version_info < (3, 10):
-            py_version = "EmitApiPython39"
-        elif sys.version_info < (3, 11):
-            py_version = "EmitApiPython310"
-        elif sys.version_info < (3, 12):
-            py_version = "EmitApiPython311"
+        py_version = "EmitApiPython"
+        if sys.version_info > (3, 9):
```

Comment:
```suggestion
        if sys.version_info > (3, 7):
```

Reply:
Since we are skipping these tests for versions that aren't supported, we won't encounter the unversioned type name and can remove this condition.